TelemetryDashboard/TelemetryDashboard.js

```diff
@@ -115,6 +271,17 @@ function setup_connect(button_svg, button_color) {
         }
 
         ws.onmessage = (msg) => {
+            if (msg.data[0] == "{") {
```

Comment:
I think you should just be able to check if its a string. Presuming that the json data is sent as a text message and that raw is sent with the binary message.